rotkehlchen/tests/api/test_evm_transactions.py

```diff
@@ -89,3 +129,112 @@ def test_query_transactions(rotkehlchen_api_server: 'APIServer'):
         for entry in result['entries']:
             assert entry['entry']['timestamp'] <= last_ts
             last_ts = entry['entry']['timestamp']
+
+
+@pytest.mark.parametrize('ethereum_accounts', [[ADDY]])
```

Comment:
This test should use vcr: It's completely repeatable (all new tests with many remote calls should): https://rotki.readthedocs.io/en/latest/contribute.html#check-and-mock-networking